ANS_JS_template.tex

```diff
@@ -93,84 +93,96 @@
+While there are definite benefits of a NRHES, there are clear challenges to implementation. The main drawback of co-locating the various generation sources and the industrial processes is the increased complexity of the system.  While there are cases of thermal coupling of nuclear power plants internationally; in Norway, Switzerland, Germany, and Canada \cite{Verfondern};  there have never been any nuclear thermal couplings thus far in the United States.  The low level state of development as well as the lack of experimental data in the United States suggest a demand for rigorous technical work.  There are concerns surrounding the regulatory oversight with a thermally coupled NRHES and whether it falls under the Nuclear Regulatory Commission's umbrella. There are multiple economic concerns. A NRHES would need contracts that ensure all of the participants in the NRHES are profitable if the system as a whole is profitable. Further policy and economics investigations need to analyze when running the NRHES in an economically optimal way is considered market manipulation. So, while there are motivations to analyze the benefits of a NRHES, those benefits always need to be analyzed with the context of the major economic, technical, and excess overhead costs also associated with a complex interconnected system.
 
-Our literature review revealed that a lot of work on NRHES is in the design and analysis stage \cite{Boardman2013, Shropshire2012}. Research is focused on developing a model that can dynamically simulate the contributions of variable energy sources, the constraints of fluctuating electricity demand, and thermal/electrical power demands for various industrial processes. The models focus on answering questions regarding minimizing the cost of electricity and ensuring profitability for each of the components comprising the NRHES. In order to move forward with development of any NRHES,  such systems must demonstrate that they can be profitable and can reliably respond to fluctuating electricity production and demand \cite{Rabiti2015}.
+The additional products include, but are not limited to: synthetic fuel, desalinated water, hydrogen production, metals production, and heat pumps \cite{Bienvenue2015}. The nuclear power plant diverts the energy it generates between the grid and the industrial process to maximize profit or minimize costs. The system can allocate energy based on the market value of each of the multiple outputs produced. In the case of a nuclear power plant coupled with a desalination plant, for example, if the price of electricity is low, more thermal and/or electric energy can be diverted to generate more clean water at a higher profit \cite {Chen2016}. The costs of running a dynamic system, such as the case with a NRHES, include additional wear and tear, decreased thermal efficiency, and increased complexity of operations \cite{Garcia2013}. The total costs of the system have yet to be determined.
```

Comment:
Have there been no economic studies yet at all? You might want to be more demonstrative in that last statement. Because if there hasn't, then we're all just basically blowing smoke, and I think we're missing a big piece of the puzzle if we seriously think we can implement these systems.